components/alternate-accessions.js

```diff
@@ -0,0 +1,26 @@
+// node_modules
+import PropTypes from "prop-types";
+
+/**
+ * // Display the alternate accessions, normally below the header line in objects.
+ */
+export default function AlternateAccession({ alternateAccessions = [] }) {
+  if (alternateAccessions?.length > 0) {
+    return (
+      <h4 className="text-gray-500">
+        {alternateAccessions.length === 1 ? (
+          <span>Alternate Accession: {alternateAccessions[0]}</span>
```

Comment:
Should this just say “Alternate: something” instead of “Alternate Accession: something”? I feel like this is really long, and it seems obviously an accession. I know ENCODE uses “Alternate accession.”

Reply:
i like Alternate Acession better than Alternate.